scripts/ci/helpers.sh

```diff
@@ -9,6 +9,10 @@ PASS() {
     echo -e "$GREEN""[PASS]""$NC" "$*"
 }
 
+INFO() {
+    echo -e "$YELLOW""[INFO]""$NC" "$*"
```

Comment:
Personally find this to look nicer.

```suggestion
    echo -e "${YELLOW}[INFO]${NC}" "$*"
```

Or alternatively just

```suggestion
    echo -e "$YELLOW[INFO]$NC" "$*"
```